desc/coils.py

```diff
@@ -716,21 +722,29 @@ class CoilSet(OptimizableCollection, _Coil, MutableSequence):
     Parameters
     ----------
     coils : Coil or array-like of Coils
-        collection of coils. Must all be the same type and resolution.
-    currents : float or array-like of float
-        currents in each coil, or a single current shared by all coils in the set
+        Collection of coils. Must all be the same type and resolution.
+    currents : float or array-like of float (optional)
```

Comment:
Don't think we need currents as an input since each coil should already have its own current